src/librustc_resolve/lib.rs

```diff
@@ -1010,7 +1010,7 @@ pub struct ModuleData<'a> {
     resolutions: RefCell<FxHashMap<(Ident, Namespace), &'a RefCell<NameResolution<'a>>>>,
     legacy_macro_resolutions: RefCell<Vec<(Ident, MacroKind, ParentScope<'a>,
                                            Option<&'a NameBinding<'a>>)>>,
-    macro_resolutions: RefCell<Vec<(Vec<Segment>, ParentScope<'a>, Span)>>,
+    macro_resolutions: RefCell<Vec<(Vec<Ident>, ParentScope<'a>, Span)>>,
```

Comment:
Oh, man.
Couldn't you just wait with beta for a day longer, as it was promised.

Reply:
@Mark-Simulacrum said they were OK with this on IRC.

Reply:
RC2 was announced in the blog to be released today, so we wanted to reduce the delay for beta.

Reply:
OK, but "resolve: Make sure macros and imports are resolved in full parent scope" was somehow cherry-picked incorrectly and "fix errors caused by cherry picking" now "fixes" something that was already correct.